src/MOIWrapper.jl

```diff
@@ -443,44 +426,44 @@ function LQOI.get_primal_status(instance::Optimizer)
     if XPR.is_mip(instance.inner)
         stat_mip = XPR.get_mip_status2(instance.inner)
         if stat_mip in [XPR.MIP_Solution, XPR.MIP_Optimal]
-            return MOI.FeasiblePoint
+            return MOI.FEASIBLE_POINT
         elseif XPR.MIP_Infeasible && XPR.hasdualray(instance.inner)
-            return MOI.InfeasibilityCertificate
+            return MOI.INFEASIBILITY_CERTIFICATE

```

Comment:
I think we should have NO_SOLUTION, because the certificate is actually in the dual

Reply:
even better, just remove this branch